release/cli/cmd/create-branch.go

```diff
@@ -0,0 +1,182 @@
+package cmd
+
+/*
+	what does this command do?
+
+	if release type is "minor" then :
+	creates a new release branch in upstream eks-a repo based off "main" & build tooling repo
+
+	creates a new release branch in forked repo based off newly created release branch in upstream repo
+
+	else :
+	creates a new patch branch in users forked repo based off latest release branch upstream
+
+*/
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/google/go-github/v62/github"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildToolingRepoName = "eks-anywhere-build-tooling"
+	upStreamRepoOwner    = "testerIbix" // will eventually be replaced by actual upstream owner, aws
+)
+
+// createBranchCmd represents the createBranch command
+var createBranchCmd = &cobra.Command{
+	Use:   "create-branch",
+	Short: "Creates new release branch from updated trigger file",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command.`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+
+		err := releaseDecision()
+		if err != nil {
+			fmt.Printf("error creating branch %s", err)
+		}
+	},
+}
+
+func releaseDecision() error {
+	RELEASE_TYPE := os.Getenv("RELEASE_TYPE")
+
+	if RELEASE_TYPE == "minor" {
+		err := createMinorBranches()
+		if err != nil {
+			fmt.Printf("error calling createMinorBranches %s", err)
```

Comment:
Let's return these errors to the caller instead of using Printf statements throughout.